FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -276,17 +405,12 @@ public function testUseShopSearch(
             ['ActivateFindologicForCategoryPages', $isActiveForCategory],
             ['IntegrationType', $checkIntegration ? Constants::INTEGRATION_TYPE_DI : Constants::INTEGRATION_TYPE_API]
         ];
-
         $request = new RequestHttp();
```

Comment:
Test case `If no request is given => should return true` has not been added, as the request seems to always be created.

Reply:
there is already a separate test for this case called `testUseShopSearchWhenRequestIsNull`